tests/integration_tests/test_hyperopt_ray_horovod.py

```diff
@@ -81,25 +81,25 @@ def mock_storage_client(path):
             "dynamic_resource_allocation": True,
         },
     },
-    {
-        "type": "ray",
-        "search_alg": {"type": "bohb"},
-        "scheduler": {
-            "type": "hb_bohb",
-            "time_attr": "training_iteration",
-            "reduction_factor": 4,
-        },
-        "num_samples": 3,
-    },
+    # {
+    #     "type": "ray",
+    #     "search_alg": {"type": "bohb"},
+    #     "scheduler": {
+    #         "type": "hb_bohb",
+    #         "time_attr": "training_iteration",
+    #         "reduction_factor": 4,
+    #     },
+    #     "num_samples": 3,
+    # },
 ]
 
 EXECUTORS = [
-    {"type": "ray"},
+    {"type": "ray", "cpu_resources_per_trial": 0.1},
```

Comment:
Why `0.1`? Could we use an integer like 1?

Reply:
Running a test cluster with 4 cpus exhausts all the resources if every trial as a full cpu.